nmcli/features/ipv6.feature

```diff
@@ -1043,20 +1043,49 @@
 
     @rhbz1083133 @rhbz1098319 @rhbz1127718
     @veth @eth1_disconnect
+    @ver-=1.11.1
     @ipv6_add_static_address_manually_not_active
-    Scenario: NM - ipv6 - add a static address manually to non-active interface
+    Scenario: NM - ipv6 - add a static address manually to non-active interface (legacy 1.10 behavior and older)
     Given "testeth1" is visible with command "nmcli connection"
     Given "eth1\s+ethernet\s+connected" is not visible with command "nmcli device"
     Given "state UP" is visible with command "ip a s eth1"
     * "0" is visible with command "cat /proc/sys/net/ipv6/conf/eth1/disable_ipv6"
     * Execute "ip -6 addr add 2001::dead:beef:01/64 dev eth1"
     Then "0" is visible with command "cat /proc/sys/net/ipv6/conf/eth1/disable_ipv6"
     Then "inet6 2001::dead:beef:1/64 scope global" is visible with command "ip a s eth1"
+    # Newer versions of NM no longer create IPv6 LL addresses for externally assumed devices.
+    # This test is obsoleted by @ipv6_add_static_address_manually_not_active_2, but this
+    # behavior won't be backported to older versions.
+    Then "addrgenmode none " is visible with command "ip -d l show eth1"
     Then "inet6 fe80" is visible with command "ip a s eth1" in "5" seconds
     # the assumed connection is created, give just some time for DAD to complete
     Then "eth1\s+ethernet\s+connected\s+eth1" is visible with command "nmcli device" in "5" seconds
 
 
+    @rhbz1083133 @rhbz1098319 @rhbz1127718
+    @veth @eth1_disconnect
+    @ver+=1.11.2
+    @ipv6_add_static_address_manually_not_active_2
```

Comment:
test case name has to be the same as pevious one 
so just:
ipv6_add_static_address_manually_not_active

version will be selected according to NM version